client/my-sites/plugins/main.jsx

```diff
@@ -633,6 +633,36 @@ export default React.createClass( {
 		return some( this.props.sites.getSelectedOrAllWithPlugins(), site => site && site.jetpack && site.canUpdateFiles );
 	},
 
+	updateAllPlugins: function() {
+		PluginsActions.removePluginsNotices( this.state.notices.completed.concat( this.state.notices.errors ) );
+		this.state.plugins.forEach( plugin => {
+			plugin.sites.forEach( site => PluginsActions.updatePlugin( site, site.plugin ) );
+		} );
+		this.recordEvent( 'Clicked Update all Plugins', true );
+	},
+
+	getNavItems() {
```

Comment:
should this function be named render instead of get? to be consistent with the rest of the functions?